docs/about_sat/view_sat_docs.md

```diff
@@ -0,0 +1,67 @@
+# View SAT Documentation
+
+You can view the System Admin Toolkit (SAT) documentation both online and
+offline as described in this section.
+
+## Online Documentation
+
+The SAT documentation can be found online in HTML form at the following link:
+[SAT Documentation](https://cray-hpe.github.io/docs-sat). The navigation pane
+on the left of the HTML page is ordered alphabetically. You can also navigate
+an individual topic's headings by using the **Headings** icon at the top of the
+page, as shown in the following images.
+
+![HTML Heading Icon](../img/HTML_Heading_Icon.png)
+
+![HTML Heading Navigation](../img/HTML_Heading_Navigation.png)
+
+The documentation can also be viewed online in GitHub by navigating to the
+`docs/` subdirectory of the
+[`docs-sat` repository](https://github.com/Cray-HPE/docs-sat/tree/integration).
+There you can navigate an individual topic's headings with a similar
+**Headings** icon at the top of the page, as shown in the following images.
+
+![GitHub Heading Icon](../img/GitHub_Heading_Icon.png)
+
+![GitHub Heading Navigation](../img/GitHub_Heading_Navigation.png)
+
+## Offline Documentation
+
+The SAT documentation is available offline as markdown, which can be
+viewed with a markdown viewer or with a text editor. The offline
+documentation is available in the `docs/` directory of the SAT release
+distribution as well as in RPM package format. The RPM package is
+installed as a part of the Ansible plays launched by the Configuration
+Framework Service (CFS). Its files are installed to `/usr/share/doc/sat`.
+
+## Check for the Latest Documentation
+
+You can find the latest documentation in the most recent RPM package. This
+package may include updates, corrections, and enhancements that were not
+available until after the software release.
+
+1. Check the version of the currently installed SAT documentation.
+
+   This RPM will only be installed if the SAT CFS configuration layer has been
+   applied.
+
+   ```bash
+   rpm -q docs-sat
+   ```
+
+2. Download and upgrade to the latest SAT documentation RPM.
+
+   ```bash
+   rpm -Uvh https://release.algol60.net/sat/docs-sat/docs-sat-latest.noarch.rpm
+   ```
+
+   If the system does not have direct internet access, you must externally
+   download this RPM and copy it to the system. The following example copies it
+   to `ncn-m001`.
+
+   ```bash
+   wget https://release.algol60.net/sat/docs-sat/docs-sat-latest.noarch.rpm -O docs-sat-latest.noarch.rpm
+   scp -p docs-sat-latest.noarch.rpm ncn-m001:/root
+   ssh ncn-m001
+   rpm -Uvh docs-sat-latest.noarch.rpm
+   ```
```

Comment:
I would suggest leaving this part just in the `rpm/README.md` since it's pretty specific to viewing offline docs and making sure they are up-to-date.

Reply:
Updated in next commit.